Update homework_3.sql

RamyaAnand27 opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
adding sql commands to the questions

## What did you learn from the changes you have made?
Joins, temp table, Aggregate functions, date function

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
no

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
no

## How were these changes tested?
in sqllite browser

## A reference to a related issue in your repository (if applicable)
NA

## Checklist
- [ Y ] I can confirm that my changes are working as intended

SidraBushra commented:
For HW3, your Aggregate query is incorrect, and for date query put month and year in group by command.

RamyaAnand27 commented:
> For HW3, your Aggregate query is incorrect, and for date query put month and year in group by command.

@SidraBushra , I have simplified the aggregate query now, though my earlier SQL was giving same result it was using a subquery which I think is not necessary. Thanks
For group by, I have added the year and month columns, though the query was working the same way without these in the group by clause :-)
